packages/integration-tests/package.json

```diff
@@ -21,10 +21,11 @@
     "start": "pnpm test"
   },
   "devDependencies": {
+    "@jest/test-sequencer": "^29.5.0",
     "@jest/types": "^29.1.2",
     "@logto/connector-kit": "workspace:^1.1.0",
-    "@logto/language-kit": "workspace:^1.0.0",
     "@logto/js": "^2.0.1",
+    "@logto/language-kit": "workspace:^1.0.0",
```

Comment:
unused?

Reply:
Hmmm, seems so.
But I didn't add it myself, it's just re-ordered. Let me try to remove it.